CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changes by Version
 ==================
 
+0.43.0
+-------------------
+* Bumped OpenTelemetry Collector to v0.43.0
+* Fix collector config update ([#670] (https://github.com/open-telemetry/opentelemetry-operator/pull/670), [@mcariapas](https://github.com/mcariapas))
+* Remove deprecate flags --metrics-level and --metrics-addr (#4695)
+  - Usages of `--metrics-level={VALUE}` can be replaced by `--set=service.telemetry.metrics.level={VALUE}`;
```

Comment:
How about removing the three lines about this change, replacing them with this:

* Upgrade to 0.43.0 will move the metrics CLI arguments into the config, in response to open-telemetry/opentelemetry-collector#4695